kolibri/plugins/user/assets/src/views/SignInPage.vue

```diff
@@ -2,192 +2,100 @@
-                <KTextbox
-                  id="username"
-                  ref="username"
-                  v-model="username"
-                  autocomplete="username"
-                  :autofocus="!hasMultipleFacilities"
-                  :label="coreString('usernameLabel')"
-                  :invalid="usernameIsInvalid"
-                  :invalidText="usernameIsInvalidText"
-                  @blur="handleUsernameBlur"
-                  @input="showDropdown = true"
-                  @keydown="handleKeyboardNav"
-                />
-              </transition>
-              <transition name="list">
-                <div class="suggestions-wrapper">
-                  <ul
-                    v-if="simpleSignIn && suggestions.length"
-                    v-show="showDropdown"
-                    class="suggestions"
-                    :style="{ backgroundColor: $themeTokens.surface }"
-                  >
-                    <UiAutocompleteSuggestion
-                      v-for="(suggestion, i) in suggestions"
-                      :key="i"
-                      :suggestion="suggestion"
-                      :style="suggestionStyle(i)"
-                      @mousedown.native="fillUsername(suggestion)"
-                    />
-                  </ul>
-                </div>
-              </transition>
-              <transition name="textbox">
-                <KTextbox
-                  v-if="needPasswordField"
-                  id="password"
-                  ref="password"
-                  v-model="password"
-                  type="password"
-                  autocomplete="current-password"
-                  :label="coreString('passwordLabel')"
-                  :autofocus="simpleSignIn"
-                  :invalid="passwordIsInvalid"
-                  :invalidText="passwordIsInvalidText"
-                  :floatingLabel="!autoFilledByChromeAndNotEdited"
-                  @blur="passwordBlurred = true"
-                  @input="handlePasswordChanged"
-                />
-              </transition>
-              <div>
-                <KButton
-                  class="login-btn"
-                  type="submit"
-                  :text="coreString('signInLabel')"
-                  :primary="true"
-                  :disabled="busy"
-                />
-              </div>
-            </form>
-
-            <p class="create">
-              <KRouterLink
-                v-if="canSignUp"
-                :text="$tr('createAccountAction')"
-                :to="signUpPage"
-                :primary="true"
-                appearance="flat-button"
-              />
-            </p>
-            <div slot="options">
-              <component :is="component" v-for="component in loginOptions" :key="component.name" />
-            </div>
-            <p
-              v-if="showGuestAccess"
-              class="guest small-text"
+    <AuthBase>
+      <div
+        v-if="hasMultipleFacilities"
```

Comment:
I think the logic should be 

1. Show the "Sign in..." prompt whenever there are more than 1 facilities _or_ this is a non-personal instance.
2. Show the "change" link only if there are 1+ facilities.